source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+
+### Scanning datasource
+
+To start scanning, press the scan button on the registered datasource.
+
+<img src="/images/azure-purview/start-scan.png" />
+
+The data that has to be filled in is pretty straightforward. Under the credentials, we can select how we want to connect to it. When setting it up for the Purview MSI, it requires extensive permissions to a point where you have to give it database owner permission on the database. This gives it full access to the database.
+
+If this is something that you dont want, you can create a new managed identity.
+
+To create a new managed identity, it requires a name, what kind of authentication method it should use (we selected SQL authentication). No matter what method you use, it will require you to use Azure Keyvault credentials which make sure you don’t hardcode the password anywhere. This is a very nice security feature they included. 
+
+<img src="/images/azure-purview/create-credential.png" />
+
+After having set up the managed identity and given the managed identity access to the keyvault, it is time to start the scan. Select which tables you want to scan and in which interval you want the scan to be executed. You can also create your own custom rulesets, that is outside the scope of this article. 
+
+<img src="/images/azure-purview/sqldatabase.png" />
+
+After the scan is complete, the details on the database pane will show how many scans have run and how many assets it has scanned. 
+
+## Data catalog
+
+After the data source registrations and scans, we can use the results of the scanned data to know more about the data in our organisation. 
+
+We can start on the homepage, which is accessible to the organisation (access can be given to accounts, based on role). When we start searching you can see that we start to see data about the database we just scanned. When typing in customer, it shows the result of the asset it found, which in this case is a table about customer addresses. 
+
+<img src="/images/azure-purview/search.png" />
+
+Once you have found something you want to know more of, you can simply click the result to show more details about the asset
+
+<img src="/images/azure-purview/metadata.png" />
+
+On the overview page, you get to see a lot of useful metadata as well as the hierarchy on the right. The hierarchy shows where the table came from. In our case, it came from a schema called SalesLT which is located in the database "cr-purview" (which we just scanned) on the server "crpurview.database.windows.net". 
+
+Something that already existed before Azure Purview was introduced as Data catalog. As described earlier, this gave a way to do scans on existing data sources, like a SQL database. It would then show what tables this database contained and what those tables contained. 
+
+Azure Purview still uses Azure Data Catalog. Data Catalog enables you to register sources, automated scanning and classification of data as well as data discovery.
```

Comment:
Data Catalog enables you to register sources, automated scanning and classification of data as well as data discovery. >> Data Catalog enables you to register sources, automate scanning, as well as discover and classify data.